### PR TITLE
Update image-update.md to allow secret creation

### PR DESCRIPTION
**Pull request by dafstone**
Patch the current documentation to allow continuation of line where required to create the docker-registry secret in the example.  Without this the example will not run due to a bash error.  

Output of error this fixes:

```
secret/ecr-credentials created (dry run)
/bin/bash: line 5: -o: command not found
error: no objects passed to apply
```

### CODE REVIEW

**Review by hiddeco — APPROVED** [review]

**Comment by hiddeco** [comment]
Thank you for the patch! Before we are able to merge we need a sign-off on your commit, https://github.com/fluxcd/website/pull/311/checks?check_run_id=2739153905 tells you how to do this.

**Comment by dafstone** [comment]
@hiddeco -- all set there, thanks for the tip -- didn't realize that was facing me.  All signed.

**Pull request merged**
